lua/conform/formatters/nufmt.lua

```diff
@@ -0,0 +1,8 @@
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://github.com/nushell/nufmt",
+    description = "The nushell formatter",
+  },
+  command = "nufmt",
```

Comment:
Do you need to pass `--stdin` to get it to format from stdin/out? The docs seem to indicate that it's required.

Reply:
Thanks for pointing out. `$FILENAME` is what is required though.

Reply:
I would expect you to also need to add `stdin = false`

Reply:
What I mean is, I would expect that either you are setting the option `stdin = false` and passing `args = { "$FILENAME" }` to indicate to the process that it should format a file, OR that you set `stdin = true` (or leave it out as that is the default) and pass `args = { "--stdin" }`. What I suspect is happening right now is that conform thinks it is running a formatter with stdin/stdout, but `nufmt` is formatting a file. That will work in simple cases, but in some situations cause very subtle bugs that are hard to track down.

Reply:
Got it! You are referring to conform option.
Fixed it!